packages/NetteToSymfony/tests/Rector/Class_/KdybyEventDoctrineSubscriberToDoctrineSubscriberRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rector\NetteToSymfony\Tests\Rector\Class_\KdybyEventDoctrineSubscriberToDoctrineSubscriberRector\Fixture;
+
+use Kdyby\Doctrine\Events;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class SomeDoctrineEventSubscriber implements EventSubscriberInterface
```

Comment:
According to the [documentation](https://github.com/Kdyby/Events/blob/master/docs/en/index.md#doctrine-events), the interface to implement is `Kdyby\Events\Subscriber`, might this be an issue or is it okay?

Reply:
I probably incorrectly copied the code in #1045

I'll change it, no problem. Thanks

Reply:
The format is completelly differnt than in the test :D I'm lost.

Could you add the proppre change example in diff? Or link to entrydo code would help, I could not find it.